ast/src/main/java/com/graphicsfuzz/common/ast/TranslationUnit.java

```diff
@@ -184,6 +194,21 @@ public TranslationUnit clone() {
         .collect(Collectors.toList());
   }
 
+  public VariablesDeclaration getUniformDeclaration(String name) {
+    return getGlobalVariablesDeclarations().stream()
+        .filter(item -> item.getBaseType().hasQualifier(TypeQualifier.UNIFORM)
+            && item.getDeclInfo(0).getName().equals(name))
+        .findAny()
+        .get();
+  }
+
+  public boolean hasUniformDeclaration(String name) {
```

Comment:
Similar comment to the above - it should handle multiple uniforms or zero uniforms in a single declaration (and you may as well add a JavaDoc comment.)

Reply:
Done.